products/paper/main.Rmd

```diff
@@ -30,7 +30,8 @@ knitr::opts_chunk$set(echo = FALSE, message = FALSE, error = FALSE, warning = FA
 
 # Introduction
 
-	Healthcare in the United States, while not restricted in variety, poses several barriers for individual access to its services. Private healthcare insurance provides the socially desired increase in choice of options, but often results in higher out of pocket costs and deductibles (Doonan et al. 2015). As a result, only the wealthy few with little to no health issues benefit. Medicare additionally provides significant choice, “but also entails significant out-of-pocket costs for its seniors and disabled beneficiaries” (Doonan et al. 2015, 755). Medicaid promises to give recipients open choice to providers, but the federal government allows states to secure waivers to instead provide a limited network (Doonan et al. 2015). Widespread use of Multiplan networks adds another node of complication to understanding access and coverage. This apparent limitless quantity of options has not, however, resulted in an increase in the quality of care recipients receive. As of 2011, “just under one-quarter of the inhabitants of the more developed regions lived in rural areas,” and while urban centers are generally assumed to be “less-healthy due to higher levels of overcrowding, air pollution, and stress,” they are in fact more healthy than their rural counterparts (Hughes 2019). Rural communities, specifically rural Indigenous communities in the United States, have unprecedented health disparities and limited regularity of healthcare (Kruse et al. 2016). 
+Healthcare in the United States, while not restricted in variety, poses several barriers for individual access to its services. Private healthcare insurance provides the socially desired increase in choice of options, but often results in higher out of pocket costs and deductibles (Doonan et al. 2015). As a result, only the wealthy few with little to no health issues benefit. Medicare additionally provides significant choice, “but also entails significant out-of-pocket costs for its seniors and disabled beneficiaries” (Doonan et al. 2015, 755). Medicaid promises to give recipients open choice to providers, but the federal government allows states to secure waivers to instead provide a limited network (Doonan et al. 2015). Widespread use of Multiplan networks adds another node of complication to understanding access and coverage. This apparent limitless quantity of options has not, however, resulted in an increase in the quality of care recipients receive. As of 2011, “just under one-quarter of the inhabitants of the more developed regions lived in rural areas,” and while urban centers are generally assumed to be “less-healthy due to higher levels of overcrowding, air pollution, and stress,” they are in fact more healthy than their rural counterparts (Hughes 2019). Rural communities, specifically rural Indigenous communities in the United States, have unprecedented health disparities and limited regularity of healthcare (Kruse et al. 2016). 
+
```

Comment:
Don't use tab indentation for paragraphs. Separate them with an empty line instead.